tests/Rector/Contrib/PHPUnit/SpecificMethodInternalTypeRector/Test.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Tests\Rector\Contrib\PHPUnit\SpecificMethodInternalTypeRector;
+
+use Rector\Rector\Contrib\PHPUnit\SpecificMethodInternalTypeRector;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+final class Test extends AbstractRectorTestCase
+{
+    public function test(): void
+    {
+        $this->doTestFileMatchesExpectedContent(
+            __DIR__ . '/Wrong/wrong.php.inc',
+            __DIR__ . '/Correct/correct.php.inc'
+        );
+    }
+
+    /**
+     * @return string[]
+     */
+    protected function getRectorClasses(): array
+    {
+        return [SpecificMethodInternalTypeRector::class];
+    }
```

Comment:
Nice test :clap:

Reply:
Working like #200, to verify custom error messages aren't overwrite.